lib/ansible/modules/pip.py

```diff
@@ -663,11 +723,34 @@ def is_satisfied_by(self, version_to_test):
                 for op, ver in self._requirement.specs
             )
 
+    @property
+    def has_requirement(self) -> bool:
+        return self._requirement is not None
+
+    @property
+    def download_information(self) -> str:
+        """Returns the necessary information to pass into pip download"""
```

Comment:
(imperative mood for methods, full sentence)
```suggestion
        """Return the necessary information to pass into pip download."""
```

Reply:
Not in code anymore.